### ISSUE TITLE
Blade is still complied 

### ISSUE THREAD

**Issue by craigpotter** [facade/ignition#100]
Hi, 

When I have an error is my blade, it is still showing the cached complied version instead of the nice neat blade behind it? 
Do I need to do something to activate this feature? 

This app was running Laravel 5.8 and was upgraded to 6.

**Comment by mpociot** [comment]
No, there shouldn't be the need to activate anything. Is there anything "special" going on in your blade views? What kind of exception do you get?

**Comment by craigpotter** [comment]
Not really. 

I tried to create a Flare Share to send you but I got:
```
We were unable to share your error.
Please try again later.
```

It is just a basic view file with some vue components in it. 
I've stripped it back a little but just doing a undefined variable in the first line breaks it. 

![image](https://user-images.githubusercontent.com/1442635/64360469-78666800-d002-11e9-99d9-7ff0c5c8fbe0.png)

**Comment by mpociot** [comment]
@craigpotter can you help me debug this a bit?

We fall back to the compiled view if we can not find the proper original blade file.
So we compare the compiled path with the paths that have been stored in the compiler: https://github.com/facade/ignition/blob/master/src/Views/Concerns/CollectsViewExceptions.php#L53-L55

Can you dump the two strings that we compare in that closure? Maybe that's an issue with symlinks or something like that.

**Comment by craigpotter** [comment]
Looks like a windows issue... 

`dd(realpath(dirname($comparePath)).'/'.basename($comparePath), $compiledPath);`

`"C:\projects\project\storage\framework\views/e98146ee15d5a38821770cf22e1eaf88eb369e47.php"`
`"C:\projects\project\storage\framework\views\4217c62df413186a96ccbbfc1b16a3ed7bd90d1a.php"`

**Comment by craigpotter** [comment]
The view is actually in 

"C:\projects\project\resources\views\config\_design.blade.php"

Moving the project in to homestead so I will test it then

**Comment by mpociot** [comment]
Ahh okay. I'll use the `DIRECTORY_SEPERATOR` constant there and it should be fixed!

**Comment by mpociot** [comment]
@craigpotter I just published version 1.4.14 that should fix this 👍

**Comment by craigpotter** [comment]
Yep that works! 👍
